.pre-commit-config.yaml

```
@@ -3,6 +3,17 @@ ci:
   autoupdate_schedule: 'quarterly'
   skip: [poetry-lock, poetry-export]
 repos:
+  - repo: https://github.com/pre-commit/pre-commit-hooks
+    rev: v4.4.0
+    hooks:
+      - id: check-added-large-files
+      - id: detect-aws-credentials
+        args: [ --allow-missing-credentials ]
+      - id: detect-private-key
+      - id: mixed-line-ending
+      - id: trailing-whitespace
+      - id: no-commit-to-branch
+        args: [--branch, main, --branch, development]
```

```suggestion
        args: [--branch, main, --branch, dev]
```
Can we just switch over to `dev`? That seemed to be the consensus in the Slack thread and is consistent with `imap_processing`.